packages/backend-tasks/src/tasks/TaskWorker.ts

```diff
@@ -51,6 +51,10 @@ export class TaskWorker {
       `Task worker starting: ${this.taskId}, ${JSON.stringify(settings)}`,
     );
 
+    const cadence = Duration.fromISO(settings.cadence);
+    const workCheckFrequency =
+      cadence < this.workCheckFrequency ? cadence : this.workCheckFrequency;
```

Comment:
Can you really compare them like this? I'd assumed you needed to compare their respective `.as('milliseconds')` or so

Reply:
This is apparently the intended way, I was searching for a `.compare(...)` or something like that 😅